src/main/resources/config.yml

```diff
@@ -35,6 +35,12 @@ db:
     elideSetAutoCommits: true
     maintainTimeStats: false
 
+# Integrations
+integrations:
+    vault:
+        # Enable Vault integration
+        enabled: false
+
```

Comment:
something like just `vault: true` without integrations or enabled IMO would be simpler